File: app/src/main/java/noahzu/github/io/gank/Base/BaseView.java
package noahzu.github.io.gank.Base;

/**
 * Created by Administrator on 2016/6/29 0029.
 */
public interface BaseView<T> {
    void showMessage(String msg);
    void showLoading();
    void hideLoading();
    void setPresenter(T presenter);
}

File: app/src/main/java/noahzu/github/io/gank/Base/BaseView.kt
package noahzu.github.io.gank.Base

/**
 * Created by zujinhao on 17/5/19.
 */
public interface BaseView<T>{
    abstract fun showMessage(msg: String)
    abstract fun showLoading()
    abstract fun hideLoading()
    abstract fun setPresenter(presenter: T)
}